javaslang-benchmark/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I need `ScalaZ`, as it's the only persistent alternative for `PriorityQueue`.

> test against other implementations such as Functional Java or PCollections instead

Good idea, added a PR for comparing performance of List's add and iteration: https://github.com/javaslang/javaslang/pull/1321